.github/workflows/ci.yml

```diff
@@ -39,7 +73,7 @@ jobs:
             ${{ steps.setup-haskell-cabal.outputs.cabal-store }}
```

Comment:
I think you can reduce this matrix to
```yaml
          # NB: PyF doesn't work with GHC <8.4 (guibou/PyF#94)
          - "8.4.1"
          - "8.6.1"
          - "8.10.1" # need an earlier revision than in Nix, because there's an API breakage that only affects 8.10.7
          - "9.2.2" # there's an API breakage introduced here
```

Reply:
done.